CONTRIBUTING.md

```diff
@@ -89,4 +89,13 @@ Note, you have to include only samples that represent the main Chainlink product
 - API functions
 - Code samples that are broken down into smaller pieces
 - Shell, bash, or text commands
-- Tables, blocks, or similar markdown structures
\ No newline at end of file
+- Tables, blocks, or similar markdown structures
+
+## Style Guide
+- **Bold** key terms or any terms the user must absolutely know when reading a doc.
+- *Italicize* terms for emphasis but only when necessary.
+- When referring to dropdown menus or a sequence of dropdown events, bold the relevant entities. Example: Go to **Menu Name** > **Option**. Do not use quotes.
+- Use markdown over HTML wherever possible.
+- When writing an additional note, start it with **Note:**.
```

Comment:
Do we have guidance for regular notes vs notes with color highlighting using the emojis? Add this link: https://rdmd.readme.io/docs/callouts